daemon/graphdriver/windows/windows.go

```diff
@@ -233,7 +233,7 @@ func (d *Driver) create(id, parent, mountLabel string, readOnly bool, storageOpt
 		}
 
 		if sandboxSize != 0 {
-			if err := hcsshim.ExpandSandboxSize(d.info, id, sandboxSize); err != nil {
+			if err = hcsshim.ExpandSandboxSize(d.info, id, sandboxSize); err != nil {
```

Comment:
Changing this to assign to an existing variable makes it inconsistent with every other two-clause `if` statement in the function. What "clean up" does this change achieve? Is it really necessary? I really don't want to verify that the variable-scoping changes here and elsewhere in the commit won't introduce any bugs without a good reason.

Reply:
I think this may have been changes I was working on to enable more linters, but I agree that for these case, it's ok to shadow the existing error; I reverted these changes.